src/core/worker.js

```diff
@@ -355,6 +355,14 @@ var WorkerMessageHandler = {
     handler.on('GetDocRequest', function wphSetupDoc(data) {
       return WorkerMessageHandler.createDocumentHandler(data, port);
     });
+
+    handler.on('GetCoverageData', function workerCoverage() {
```

Comment:
This should be reverted back to `ReportCoverageData`, since the handler handles the message that you send from here:
https://github.com/shikhar-scs/pdf.js/blob/beb7bd963db27adf6dac5d614002796c4a4de044/src/display/api.js#L1543-L1547

While you are at it - change `workerCoverage` to `wphReportCoverageData` for consistency with existing code.